### ISSUE TITLE
Request

### ISSUE THREAD

**Issue by Kristian-12345** [Edzelf/ESP32-Radio#372]
Hello Edzelf,

I am not sure if this is the correct way to contact you.

Thanks for the great project.

I found a very interesting board which contains a ESP32 and a audio decoder.

ESP32-A1S Wi-Fi+BT Audio Development Kit 

It can be found for example here: https://www.exp-tech.de/plattformen/esp32/9181/esp32-a1s-wi-fi-bt-audio-development-kit

  Do you know if your sketch will run on this board, or if it can be adapted to it?

Thanks again

Kristian

**Comment by Edzelf** [comment]
Yes and no.  The sketch will run on this board, but you still need a VS1053 for decoding the MP3 stream.  The audio decoder on the board can not decode MP3.
